### PR TITLE
Improvement/TK23 Develop Support to Dynamic Types on Registration Screen

### LINKED ISSUE #46: [IMPROVEMENT] US 1 | TK 23 Develop Support Dynamic Types on Registration Screen [3]
# Develop Support Dynamic Types on Registration Screen ✏️ 

# Description 📝
## What could be improved ✅
The Registration Screen must be able to adapt itself to the Dynamic types whenever the user wants

**Additional context** 🌎

### PR DESCRIPTION
**Pull request by illuminatyy**
# What was done? ✅
> Describe in a few words what have you done? 
> - Step 1: Created a `@Environment(\.sizeCategory) `  to show a scroll view when dynamic types was used
> - Step 2: Hidded the `navigation bar` (adjusted content on the middle of the screen)
> - Step 3: Created a var to organized Tokens Spacing on `TermsOfUseSheetScreen.swift`

# Description 📝
  
# Evidence 🕵️‍♀️
### IOS 15 (iPhone 13 Pro Max & iPod touch):
| none                                                                                                                                   | with both and wrong                                                                                                                    |
|----------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------|
| <video src="https://user-images.githubusercontent.com/51174686/179272598-c784ae81-a571-4952-beb2-e07d3bc2bffb.mp4" width=300 controls> | <video src="https://user-images.githubusercontent.com/51174686/179272811-066282ba-be5f-4969-a7cd-f90163cfa2fa.mp4" width=300 controls> |

### IOS 14 (iPhone 11 Pro Max & iPod touch):
| none                                                                                                                                   | with both and wrong                                                                                                                    |
|----------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------|
| <video src="https://user-images.githubusercontent.com/51174686/179273104-9f6ae0ce-e47a-4d8c-b4af-1bef3f11f36a.mp4" width=300 controls> | <video src="https://user-images.githubusercontent.com/51174686/179273198-cc536453-e1c7-4869-b89f-f0885995dcab.mp4" width=300 controls> |

Closes #46

### CODE REVIEW

**Review by jgabrielbiazus — APPROVED** [review]

**Review by MarinaFX — APPROVED** [review]

File: Fiero/Views/Login/RegistrationScreenView.swift [hunk 1]
```diff
@@ -112,31 +126,54 @@ struct GlassPhormism<Content>: View where Content: View {
...
+            if dynamicTypeCategory > .extraExtraLarge  {
+                ScrollView {
+                    content
+                        .frame(width: UIScreen.main.bounds.width * 0.9)
+                        .padding(.vertical, Tokens.Spacing.xxs.value)
+                }
+            } else {
+                ZStack {
+                    Image("LoginBackground")
+                        //.resizable()
```
Comment (MarinaFX) [inline Fiero/Views/Login/RegistrationScreenView.swift:165]: remover codigo comentado

**Review by MarinaFX — CHANGES_REQUESTED** [review]
so remover o codigo comentado ali e tudo top

File: Fiero/Views/Login/RegistrationScreenView.swift (hunk 1 above)
Comment (illuminatyy) [reply Fiero/Views/Login/RegistrationScreenView.swift]: Feito!!

**Review by MarinaFX — APPROVED** [review]
approved

**Pull request merged**